Shared/AppSecrets.swift.masque

```diff
@@ -15,6 +15,6 @@
 import ArcGIS
 
 extension APIKey {
-    /// The default API key.
-    static let `default` = APIKey("<#API Key#>")
+    /// The API key for toolkit examples and testing.
```

Comment:
If you want this to be project agnostic, you'll want to change this comment. Perhaps something like "The API key for development of packages/projects in this repository."